code/modules/mob/living/carbon/slime/powers.dm

```diff
@@ -86,8 +86,9 @@
 			if(victim)
 				victim.updatehealth()
 
-			if(nutrition == get_max_nutrition())
+			if(nutrition >= get_max_nutrition())
```

Comment:
Is nutrition not prevented from going above max_nutrition? Or is this just an "incase" measure.

Reply:
this is just in case